src/sentry/issues/endpoints/group_details.py

```diff
@@ -299,21 +307,21 @@ def get(self, request: Request, group: Group) -> Response:
             data.update({"participants": participants})
 
             metrics.incr(
-                "group.update.http_response",
```

Comment:
All methods used the `group.update` prefix, thus, they were not distinguishable on Datadog.